### PR TITLE
Ported build from oasis to jbuilder.

### PR DESCRIPTION
**Pull request by kc284**
Signed-off-by: Konstantina Chremmou <konstantina.chremmou@citrix.com>

### CODE REVIEW

**Comment by robhoes** [comment]
Looks good to me, but travis failed...

File: vhd-tool.opam [hunk 1]
```diff
@@ -8,14 +8,9 @@ tags: [
...
 ]
-build: [
-  ["oasis" "setup"]
-  ["./configure" "--bindir=%{bin}%" "--etcdir=%{etc}%" "--libexecdir=%{bin}%"]
-  [make]
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]
 ]
 depends: [
-  "oasis" {build}
-  "ocamlfind" {build}
```
Comment (mseri) [inline vhd-tool.opam]: You should add `"jbuilder" {build & >= "1.0+beta10}` in the depends section

**Comment by kc284** [comment]
Any idea why travis is still failing even after removal of the configure line?

File: .travis.yml [hunk 2]
```diff
@@ -2,11 +2,12 @@ language: c
...
 install:
     - wget https://raw.githubusercontent.com/xenserver/xenserver-build-env/master/utils/travis-build-repo.sh
-script: bash travis-build-repo.sh
+script:
+    - cp vhd-tool.opam opam && bash travis-build-repo.sh
 sudo: true
 env:
     global:
         - REPO_PACKAGE_NAME=vhd-tool
-        - REPO_CONFIGURE_CMD=./configure
```
Comment (gaborigloi) [inline .travis.yml]:
```
REPO_CONFIGURE_CMD=true
```
should work.

Comment (kc284) [reply .travis.yml]: It seems it's passing with empty string, not with true 🤔

**Comment by kc284** [comment]
I'm running one more travis job to be sure.

**Comment by kc284** [comment]
Ok, green again with the empty string.

**Pull request merged**